Documentation~/menu.md

```diff
@@ -1,24 +1,172 @@
+
+## Edit
+
+Edit shapes.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **Edit PolyShape** | Edit a poly shape you created with **Tools** > **ProBuilder** > **Editors** > **New PolyShape**.  |
+| **Edit shape** |  Edit a shape you created with **Tools** > **ProBuilder** > **Editors** > **New Shape**. |
+
+## Dimensions Overlay
+
+Show or hide the dimensions for all three axes. 
+
+This overlay appears on all mesh objects, not just ProBuilder meshes.
+
+## Selection
+
+Change your selections.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **Grow Selection** | Expand the selection outward to adjacent faces, edges, or vertices. For more information, refer to [Grow Selection](Selection_Grow.md). |
+| **Select Hole** | Select all elements along the selected open vertex or edge. For more information, refer to [Select Holes](Selection_SelectHole.md). |
+| **Select Loop** | Select an edge loop from each selected edge in Edge editing mode, or a face loop from each selected face in Face editing mode. |
+| **Select Material** | Select all faces that have the same material as the selected face(s). For more information, refer to [Select by Material](Selection_SelectByMaterial.md). |
+| **Select Ring** | Select a ring from each selected edge in Edge editing mode, or a face ring from each selected face in Face editing mode. |
+| **Select Smoothing Group** | Select all faces that belong to the currently selected smoothing group. For more information, refer to [Select Smoothing Group](Selection_SmoothingGroup.md). |
+| **Select Vertex Color** | Select all faces on this object that have the same vertex color as the selected face. For more information, refer to [Select by Colors](Selection_SelectByVertexColor.md). |
+| **Shrink Selection** | Remove the elements on the perimeter of the current selection. For more information, refer to [Shrink Selection](Selection_Shrink.md). |
+
+## Interaction
+
+Interact with your selections.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **Toggle Drag Rect Mode** | To limit the drag selection to elements that are fully inside the drag rectangle, set the **Rect** action to **Complete**. For more information, refer to [Rect](Selection_Rect_Intersect.md). |
+| **Toggle Drag Selection Mode** | To select any elements that intersect with the drag rectangle, set the **Rect** action to **Intersect**. For more information, refer to [Rect](Selection_Rect_Intersect.md). |
+| **Toggle Handle Orientation** | Toggle between the three orientation states for Scene handles (**Global**, **Local**, or **Normal**). For more information, refer to [Orientation](HandleAlign.md). |
+| **Toggle Select Back Faces** | Define whether drag selection selects or ignores hidden elements. For more information, refer to [Select Hidden](Selection_SelectHidden.md). |
+| **Toggle X Ray** | Show or hide any selected hidden geometry. For more information, refer to [Selection X-Ray](preferences.md#sel-xray). |
+
```

Comment:
```suggestion

**Note:** The **Toggle Drag Selection Mode** option is now in the **Tool Settings** overlay.

```

Reply:
I'm on the fence about this one. We're really violating the timeless documentation principle here. However, users *do* need to know...